tfl-status/lib/plugin.js

```diff
@@ -1,5 +1,6 @@
 const plugin = require('dashboard-plugin')
 const generateViewState = require('./data-source')
+const secrets = require('./api-secrets.json');
```

Comment:
In case this isn't only done for testing :-1: this breaks the plugin :cry: 

The configuration values should come from a `config.json` when using the `plugin-tester` to run locally.

Reply:
can do

Reply:
the `configuration` is how we pass the values from admin dashboard to the plugin and how we're able to achieve multiple instances from the same deployment